api/src/main/java/org/openmrs/module/reporting/report/ReportRequestDTO.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.reporting.report;
+
+import java.util.List;
+
+public class ReportRequestDTO {
+
+  private List<ReportRequest> reportRequests;
+
+  private long reportRequestCount;
+
+  public ReportRequestDTO(List<ReportRequest> reportRequests, long reportRequestCount) {
+    this.reportRequests = reportRequests;
```

Comment:
Can't we just get the reportRequestsCount from reportRequests.size()? Or can we change the variable names to make things clearer? 😊